lib/flipper/adapters/active_record.rb

```diff
@@ -135,7 +135,9 @@ def get_all
           rows_query = features.join(gates, ::Arel::Nodes::OuterJoin)
             .on(features[:key].eq(gates[:feature_key]))
             .project(features[:key].as('feature_key'), gates[:key], gates[:value])
-          gates = @feature_class.connection.select_rows(rows_query)
+          select_method = rails_version_over_4_2? ? :select_rows : :select_all
```

Comment:
```suggestion
          select_method = respond_to?(:select_rows, true) ? :select_rows : :select_all
```

Instead of version detection, can this use feature detection here?